mev_inspect/compound_liquidations.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, List
+from web3 import Web3
+from mev_inspect.schemas.classified_traces import ClassifiedTrace
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.classified_traces import Protocol
+from mev_inspect.historical_price import get_erc20_token_decimals
+from mev_inspect.abi import get_raw_abi
+import os
+
+rpc = os.getenv("RPC_URL")
+
+if rpc is None:
+    raise RuntimeError("Missing environment variable RPC_URL")
+else:
+    w3 = Web3(Web3.HTTPProvider(rpc))
+
+V2_COMPTROLLER_ADDRESS = "0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B"
+V2_C_ETHER = "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5"
+WETH_ADDRESS = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+
+# cToken=>Token mapping (cDAI=>DAI)
+# useful for finding out the underlying asset address of the cToken seized
+def get_all_comp_markets():
```

Comment:
How does this work for backfills? Do they ever get changed between blocks?

Also can you add a return type